conduit-combinators/Data/Conduit/Combinators.hs

```diff
@@ -155,6 +155,8 @@ module Data.Conduit.Combinators
     , concatMapAccum
     , intersperse
     , slidingWindow
+    , chunksOfE
```

Comment:
Can you add `chunksOfCE` and `chunksOfExactlyCE` to the `Conduit` module as well?

Reply:
Ok, this one I am not sure I did correctly. Please check.

Reply:
That's perfect.